health_check/views.py

```diff
@@ -23,14 +25,20 @@ def get(self, request, *args, **kwargs):
         def _run(plugin):
             plugin.run_check()
             try:
-                return plugin.errors
+                return plugin.errors, plugin
```

Comment:
Is there a reason we don't just return ``plugin`` here? Either way we're changing the method signature, but given it's declared within the scope of the view itself, no one else should be relying on it. Seems cleaner that way.

Reply:
You're absolutely right, thanks!